napari/_qt/widgets/qt_extension2reader.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, parent=None, npe2_readers=None, npe1_readers=None):
                 'Available readers will be filtered to those compatible with your pattern. Hover over a reader to see what patterns it accepts.'
             )
             + trans._(
-                '\n\nPreference saving for folder readers is not supported, so these readers are not shown.'
+                '\n\nYou can save a preference for a specific folder by listing the folder name with a "/" at the end (for example, "/test_images/").'
             )
             + trans._(
                 '\n\nFor documentation on valid filename patterns, see https://docs.python.org/3/library/fnmatch.html'
```

Comment:
all these trans strings should be one call as translation should contain the whole context.